test/forms/event20A/BecameDateFormProviderSpec.scala

```diff
@@ -34,18 +34,6 @@ class BecameDateFormProviderSpec extends SpecBase
 
   ".becameDate" - {
 
-    behave like mandatoryDateField(
-      form = form,
-      key = becameDateKey,
-      requiredAllKey = "genericDate.error.invalid.allFieldsMissing"
-    )
-
-    behave like dateFieldYearNot4Digits(
-      form = form,
-      key = becameDateKey,
-      formError = FormError(becameDateKey, "genericDate.error.invalid.year")
-    )
-
```

Comment:
We can probably add these back in because these `behave like mandatoryDateField` and `behave like dateFieldYearNot4Digits` tests are still present in other specs

Reply:
Those are duplicates event20/BecameDateFormProviderSpec.scala has the same tests.

Reply:
Resolved after discussion: duplicated tests